core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/customize-interaction-modal.template.html

```diff
@@ -30,14 +30,14 @@ <h3 ng-if="StateInteractionIdService.displayed && (customizationModalReopened ||
   <form ng-if="StateInteractionIdService.displayed && hasCustomizationArgs" name="form.schemaForm" class="interaction-editor-form protractor-test-interaction-editor">
     <div ng-repeat="customizationArgSpec in customizationArgSpecs track by $index">
       <div ng-if="customizationArgSpec.schema.type !== 'bool'" class="oppia-interaction-customization-label"><[customizationArgSpec.description]></div>
-      <h5 ng-if="StateInteractionIdService.displayed == 'DragAndDropSortInput'">
-        <b>Note:</b> Please enter at least two choices.
-      </h5>
       <schema-based-editor ng-class="{'boolean-checkbox': customizationArgSpec.schema.type === 'bool'}" local-value="StateCustomizationArgsService.displayed[customizationArgSpec.name].value" schema="customizationArgSpec.schema">
       </schema-based-editor>
       <div ng-if="customizationArgSpec.schema.type === 'bool'" class="oppia-interaction-customization-label"><[customizationArgSpec.description]></div>
       <div style="margin-bottom: 15px;"></div>
     </div>
+    <div class="oppia-form-error oppia-units-input-error">
+      <[getCustomizationArgsWarningsList()[0].message]>
```

Comment:
Why is [0] being used? Is it always single valued list?

Reply:
No, it won't always be a single-valued list, but this would just show the first warning from the warnings list. Also, this should (but for some reason doesn't) throw a console error if the warnings list is empty. This is why I've added a validation now.

Reply:
Hi @iamprayush, I'm a bit confused about this. If there are more (>1) warnings, shouldn't they be shown as well?

Reply:
@kevintab95 I thought this might be more user friendly. Also, I haven't seen multiple warnings being displayed anywhere else in the site so I presumed that this might be the norm. This would still force the creator to resolve all the errors but it would just be one warning at a time. It seems less intimidating from the creator's POV.
But, if you disagree, I'll make the changes so that all the warnings are displayed simultaneously.

Reply:
We usually show a list of warnings in a tooltip in the editor nav bar (when you hover over the alert icon) but looks like that wouldn't work in this case because we don't allow saving of the customization args unless all the warnings are resolved. I'm fine with showing one error at a time. Thanks!